### DISCUSSION TITLE
Make seamless jobs for civilians

### DISCUSSION THREAD

**Discussion by Friiize** [EveronLife/EveronLife#56] [Ideas]
Let's add a core civilians jobs ! 

What i mean by that is to be able to pick up license work like taxi and have the ability to answer such call. In the past, we had specific jobs assign to whitelisted players (Paramedic for example), what about doing it so anyone can choose a job of their desire ? 

Of course, we must limit the power of this, we don't want player to abuse it by pretending to be a partamedic just to rob the person and so one (could be a good rp tho) because he can detect where are the players to rescue. So maybe just an indicator of how many players request a paramedic, a taxi, etc. And then, do something like a cooldown or whatever to assure the player get what he ask for. 

Anyway, so the goal would be to make a core civilian job where we can add more jobs in the futur. Any suggestions ?

**Comment by thebonbon** [comment]
Great idea.
I think limiting power of these jobs should be part of individual servers not the framework. (Server rules like "Cant rob while in Taxi job")

**Reply by Friiize** [reply]
Yup, just wonder how we can make a setting file without making a special binding to open up. Should be a list of the addons in a admin panel and in that we can access their settings so that they can be changed easily

**Comment by jasper-one** [comment]
I like this idea a lot. Particularly if you can have multiple characters on a server so you could technically train up to be the police commissioner but then get kicked out of the force for taking bribes while retaining the same character. 

To not have specific job/role slots would make the game alot more dynamic. Looking forward to discussing this more.

**Reply by Jaccodouma** [reply]
I do love the idea of cops being more actual civvies with a job rather than a whole new faction that is supposed to fight with the civvies.

**Reply by Friiize** [reply]
Well, i can code, you seems to have colaborate on something too. I'm ok to dig this with someone, i've just never contributed to a project like this so any help is welcome.

**Reply by Friiize** [reply]
@Jaccodouma Yup, i'm taking this idea from DarkRP and GTA RP, wich there was a special event between french streamers for 2 weeks and they played their role more like it and could be corrupt or something. Was really fun and would love to see something like that. 

Could be even more RP to have a cop as undercover rebel that would requier him to hide his face when he play the rebel part. It had so much complexity to it and fun.

**Reply by Arkensor** [reply]
> I do love the idea of cops being more actual civvies with a job rather than a whole new faction that is supposed to fight with the civvies.

Yep, we want to see if we can make jobs be more dyanmic. so one day you go on duty as taxi driver, the other you might be working as a tow truck driver, the other might be as medic etc. And if you want to separate your RP chars you just create multiple on the server to avoid using your RP name as cop and as civ.

**Reply by Friiize** [reply]
Like a character list ? Would that be more difficult ? I think this can be negligate at this point. You use the same character to do RP with just the possibility to play one job or another. Would be more interesting than selling fruits to buy AKs.
